.php_cs.dist

```diff
@@ -0,0 +1,27 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+     ->in(__DIR__.'/{src,tests}')
+;
+
+$header =<<<'HEADER'
+This file is part of aakb/kontrolgruppen.
```

Comment:
Should be `aakb/kontrolgruppen-core-bundle`.

Reply:
Done